### ISSUE TITLE
Loading label moves with scrolling

### ISSUE THREAD

**Issue by isaranova** [oVirt/ovirt-web-ui#727]
When I can scroll in the VM portal and I press refresh, the loading label moves with me scrolling down and up.

Tested in: ovirt-web-ui-1.4.2.0.20180814git2ec0a1e-1.fc28.noarch.rpm, Mozilla Firefox for Fedora version 61.0.1 (64-bit) 

![screenshot from 2018-08-16 13-06-12](https://user-images.githubusercontent.com/39263962/44266381-a677df00-a22a-11e8-8339-1d0d96d8de3e.png)

**Comment by gregsheremeta** [comment]
I've never liked this label and I think it should be removed.

@lizsurette @lwrigh @michalskrivanek ? webadmin does not have this.

**Comment by michalskrivanek** [comment]
it shouldn't scroll like that for sure.
The reason it's there is to know that a query is in progress and the content will reload/change. API is much slower than webadmin/backend so it could be there for quite some time
We could do a spinning wheel somewhere or anything else...but we should have an indication that something is going on, I think

**Comment by lwrigh** [comment]
@gregsheremeta @michalskrivanek @lizsurette 
PatternFly has a loading state that features a "loading" message and a spinner. https://www.patternfly.org/pattern-library/communication/loading-state/

**Comment by michalskrivanek** [comment]
yeah, but this would need to be in the background somewhere

**Comment by lizsurette** [comment]
@gregsheremeta @michalskrivanek @lwrigh Yeah, the screenshot makes it look like it's sitting on top of an already loaded screen. What is it actually still loading at this point? The pattern that Laura linked to is for when the content hasn't been loaded yet to let the user know to just hang tight. I'd be in favor of removing this loading message since it looks like to a user the cards are already loading. I'm sure there are technical reasons it's still there, but maybe we could improve that?

**Comment by gregsheremeta** [comment]
I really, really think it should just be removed :smile:

**Comment by gregsheremeta** [comment]
> What is it actually still loading at this point? 

"refreshing" would be a more appropriate term. (But again, I feel strongly that it's just a distraction, unnecessary, and should be removed. All apps refresh in the background now.)

**Comment by lwrigh** [comment]
I think as long as there's some form of content that shows up almost immediately on the page getting rid of it entirely should be fine.

**Comment by michalskrivanek** [comment]
it could be a refresh or lazy loading of icons in this case. 

It doesn't have to be so prominent...but because of the really slow response of API(say, 5s+) how else you'd show that we're waiting on something or something is still going on?

**Comment by lizsurette** [comment]
> It doesn't have to be so prominent...but because of the really slow response of API(say, 5s+) how else you'd show that we're waiting on something or something is still going on?

Right, if we do choose to show something I think it should be pinned to the top of the browser and out of the way so that the user isn't distracted and could still act on any items that have already loaded that they may want to click on. I'm in support of either removing it or keeping it and moving it out of the way (and not having centered and scrolling with the user. Similar to GMail.)

**Comment by gregsheremeta** [comment]
Gmail's are more like action notifications though -- our equivalent would be toasts. Gmail doesn't show "refreshing" every time it background fetches your mail.

**Comment by michalskrivanek** [comment]
I would still love to have something similar what browsers have somewhere in the corner. The concern is that on large,slow,remote setups this will really take tens of seconds to show something
You would need to replace that in all places with some way of showing what laura posted earlier. Which sounds to me as quite a lot of work vs a generic spinner on every api query

**Comment by lizsurette** [comment]
> Gmail's are more like action notifications though -- our equivalent would be toasts. Gmail doesn't show "refreshing" every time it background fetches your mail.

I'm referring to the loading notification rather than the toast like messages with options to undo. Here's a screenshot:
![image](https://user-images.githubusercontent.com/208455/44270257-f5ab1900-a204-11e8-89e1-f8aad9ed835b.png)
